src/test/regress/expected/single_node.out

```diff
@@ -1757,10 +1756,10 @@ BEGIN;
 (1 row)
 
 	-- now, we should have additional 32 connections
-	SELECT pg_catalog.get_all_active_client_backend_count();
- get_all_active_client_backend_count
+    SELECT count(*) FROM pg_stat_activity WHERE application_name LIKE 'citus_internal%';
```

Comment:
Now `get_all_active_client_backend_count` doesn't have any callers. 

I think it is a slight variation of the isolation test you added if we keep calling `get_all_active_client_backend_count()` such that we show we can properly count external clients -- which is only this session at this point of the code --.

So, can't we keep `get_all_active_client_backend_count()` call as well?